ts/parsers/paragraphparser.ts

```diff
@@ -65,7 +66,7 @@ export default class ParagraphParser {
         }
         let contents = textElements.map((txtElement) => {
             const content: Content = {
-                text: txtElement["a:t"] || "",
+                text: sanitizeElement(txtElement["a:t"]) || "",
```

Comment:
This needs to be done after the content has been restructured, as you see below there is a function called `restructureContents`, in that function there is an edge case handling to make the hyperlink text go into the same content string for some reasons that I can explain in person if required.
So for now if the script tag is hyperlinked, it won't replace it always.
Also can you please check as some text can be at different index in the content array even with the same word, so there is a possibility that script tag will not be on the same index of the array but rather splitted into different indexes. As each content has its properties and microsoft puts them in different indexes with their own properties for example based on formatting and such.
For example: if script tag is like this <sc**ript**/> It will put them in different indexes, same goes with italics, underlines and all those sort of things which basically change the formatting of the text in the mid way, and there are some other factors as well.
So after doing this change can you please test with the couple of examples like that and see if you are getting the desired outcome